src/layout/Grid.js

```diff
@@ -137,16 +126,238 @@
+        
+        var top = this._offsetY + ((this._currLoc[1]) * this._rowSize);
+        var left = this._offsetX + ((this._currLoc[0]) * this._colSize);
+        var width = this._colSize - this.gutter();
+        var height = this._rowSize - this.gutter();
+        
+        var handlesInRange = [];
+        if(top + height >= e.clientY && top + height - handleSize <= e.clientY && onSouthEdge){
+            handlesInRange.push('s');//within SOUTH handle range
+        }
+        else if(top <= e.clientY && top + handleSize >= e.clientY && onNorthEdge){
+            handlesInRange.push('n');//within NORTH handle range
+        }
+        if(left + width >= e.clientX && left + width - handleSize <= e.clientX && onEastEdge){
+            handlesInRange.push('e');//within EAST handle range
+        }
+        else if(left <= e.clientX && left + handleSize >= e.clientX && onWestEdge){
+            handlesInRange.push('w');//within WEST handle range
+        }
+        if(handlesInRange.length){
+            handle = handlesInRange.join('');
+        }
+        return handle;
+    };
+    
+    
+    
+    var hideDragHandles = function(){
+        var handles = document.getElementsByClassName('dragHandle');
+        for(var i in handles){
+            if(handles[i] !== null && typeof (handles[i]) !== 'undefined'){
+                if(handles[i].style !== null && typeof (handles[i].style) !== 'undefined'){
+                    handles[i].style.visibility = 'hidden';
+                }
+            }
+        }
+    };
+    
+    var showDragHandles = function(){
+        var handles = document.getElementsByClassName('dragHandle');
+        for(var i in handles){
+            if(handles[i] !== null && typeof (handles[i]) !== 'undefined'){
+                if(handles[i].style !== null && typeof (handles[i].style) !== 'undefined'){
+                    handles[i].style.visibility = 'visible';
+                }
+            }
+        }
+    };
+    
+    Grid.prototype.createDropTarget = function (loc) {
+        var context = this;
+        var col = loc[0] - this._dragCellOffsetX;
+        var row = loc[1] - this._dragCellOffsetY;
+        var colSpan = this._dragCell.gridColSpan();
+        var rowSpan = this._dragCell.gridRowSpan();
+        
+        var dropTarget = document.createElement('div');
+        dropTarget.id = 'grid-drop-target';
+        
+        setTimeout(function(){
+            document.getElementsByTagName('body')[0].appendChild(dropTarget);
+            context.updateDropTarget(col,row,colSpan,rowSpan);
+        },0);
+    };
+    
+    Grid.prototype.updateDropTarget = function (col,row,colSpan,rowSpan) {
+        var top,left,width,height;
+        top = this._offsetY + (row * this._rowSize);
+        left = this._offsetX + (col * this._colSize);
+        width = colSpan * this._colSize - this.gutter();
+        height = rowSpan * this._rowSize - this.gutter();
+        
+        setTimeout(function(){
```

Comment:
There were some issues with either Firefox (or maybe it was IE). It wasn't successfully calling "drag". At the time I thought it was caused by the grid-drop-target overlaying the dragged element too quickly. This may not be the case, I'll try removing the timeouts.